src/DigitalTwin/pnp/src/api/apiProvider.ts

```diff
@@ -10,21 +10,32 @@ import { UIConstants } from "../view/uiConstants";
  * Api provider for extension integration
  */
 export class ApiProvider {
-  constructor(private readonly modelRepositoryManager: ModelRepositoryManager) {}
+  constructor(
```

Comment:
it looks like the max length = 120 doesn't take effect since code in pnp folder is format by tslint with 120 length

Reply:
Fixed.